tests/integration/tier0/verify-latest-kernel-check-with-failed-repoquery/test_verify_kernel_check_passes.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+
+@pytest.mark.latest_kernel_check_with_failed_repoquery
+def test_verify_latest_kernel_check_passes_with_failed_repoquery(shell, convert2rhel):
+    """
+    This test verifies, that failed repoquery is handled correctly.
+    Failed repoquery subsequently caused the latest kernel check to fail.
+    Introduced fixes should get the process past the latest kernel check.
+    """
+
+    repopath = "/etc/yum.repos.d/tainted_repo.repo"
+    tainted_repo_setup = "[tainted-repo]\nname=Tainted repository\nbaseurl=http://download.eng.bos.redhat.com/beakerrepos/client/Fedor36/\nenabled=1\ngpgcheck=0"
```

Comment:
I would create a separate file called something like "broken_repo.repo" (maybe not use the word tainted, but that just my personal preference, so you can keep it :D) as a part  of this test folder instead of having the string here in the pytest file, that is hard to read. In the pytest just copy/move the file to the "/etc/yum.repos.d/". When the test will run, tmt should copy all the files to the CWD.

Reply:
You're right, I will fix that, thanks